kore/src/Kore/Step/Simplification/Simplify.hs

```diff
@@ -422,7 +422,9 @@ lookupAxiomSimplifier termLike = do
                 App_ symbol _
                   | isDeclaredFunction symbol -> do
                     let hooked = criticalMissingHook symbol
-                        unhooked = warnFunctionWithoutEvaluators symbol
+                        unhooked
+                          | noEvaluators symbol = return ()
+                          | otherwise = warnFunctionWithoutEvaluators symbol
```

Comment:
Could you please move the `noEvaluators` check inside the `warnFunctionWithoutEvaluators` function?